models/cable/NAMESPACE

```diff
@@ -0,0 +1,7 @@
+# Generated by roxygen2: do not edit by hand
+
+export(met2model.MODEL)
+export(model2netcdf.MODEL)
+export(read_restart.ModelName)
+export(write.config.MODEL)
+export(write_restart.ModelName)
```

Comment:
Did you run ```make document```? I can see below that the Rd files were generated, but am a bit surprised that all these exports are still for the 'template' versions